ninja-standalone/src/main/java/ninja/standalone/NinjaJetty.java

```diff
@@ -113,29 +123,45 @@ public NinjaJetty setNinjaContextPath(String ninjaContextPath) {
         this.ninjaContextPath = ninjaContextPath;
         return this;
     }
+	
+	public NinjaJetty setToUseWebappContext(boolean isToUse) {
+
+        this.isUseWebappContext = isToUse;
+        return this;
+    }
     
     public void start() {
 
         server = new Server(port);
+		
+        if(!isUseWebappContext) {
+            isUseWebappContext = tryToGetToUseWebappContextFromSystemProperty();
+        }
 
         try {
             
-            context = new ServletContextHandler(server, ninjaContextPath);
-            
-            NinjaPropertiesImpl ninjaProperties 
-                    = new NinjaPropertiesImpl(ninjaMode);
-            // We are using an embeded jetty for quick server testing. The
-            // problem is that the port will change.
-            // Therefore we inject the server name here:
-            ninjaProperties.setProperty(NinjaConstant.serverName, serverUri.toString());
-
-            ninjaServletListener.setNinjaProperties(ninjaProperties);
-
-            context.addEventListener(ninjaServletListener);
-
-            context.addFilter(GuiceFilter.class, "/*", null);
-            context.addServlet(DefaultServlet.class, "/");
-
+            if(isUseWebappContext) {
+                System.setProperty(NinjaConstant.MODE_KEY_NAME, ninjaMode.toString()); // set mode for Webapp context
+                String rootPath = Server.class.getClassLoader().getResource(".").toString();
+                context = new WebAppContext(rootPath + "../../src/main/webapp", ninjaContextPath);
```

Comment:
This will only work for tests and superDevMode. Not in production. Is that what you want?